src/watchful/client.py

```diff
@@ -206,8 +206,9 @@ def _read_response(
     # One idea:
     # if ret["error_msg"]:
     #     raise Exception(ret["error_msg"])
-    if "error_msg" in ret and ret["error_msg"]:
-        print(ret["error_msg"])
+    # Another idea:
+    # if "error_msg" in ret and ret["error_msg"]:
+    #     print(ret["error_msg"])
```

Comment:
I think if there is an error from the backend, it should be printed to the user using the SDK without the user explicitly seeking for it. Printing would be the most direct form of discovery on their notebooks that their API invocation will provide if there is an error from the backend that is non-crashing. Precisely because it is an error coming from the backend but not crashing the backend, it should be actively printed to tell that there is something not right. One way would be to print the backend's error message with some informative message rather than to just print the backend's error message.

Reply:
We should revert this then. I think the only problem is for me, in this test, I don't want to spam the terminal with these messages (and since the API calls are randomly generated, the error messages are not interesting to anyone either).

I'll look in the test to redirect stdout before making the API calls.

Reply:
I made another unrelated change here, while putting this back (added the verb).

Reply:
In the other test I was able to just temporarily redirect `sys.stdout` before making the API call, which solves the problem I had.